.github/workflows/validate.yml

```diff
@@ -24,15 +24,20 @@ jobs:
       checks: write
     runs-on: ubuntu-latest
     steps:
-      - if: github.event.repository.fork == false
+      - if: >-
```

Comment:
What does the `>-` do?

Reply:
It's a header for collapsing the multiline string: https://yaml-multiline.info/